### PR TITLE
#8 Feature : Sign in to Phantom

### PR DESCRIPTION
**Pull request by benshyaka**
#### What does this PR do?

- Failure and success redirect upon login
- feeedback messages when wrong email oor password is submitted

#### Description of Task to be completed?

- the users are able to sign in to app by providing their email & password.
- The administrator should be able to sign in to the app using pre-configured login credentials.

#### How should this be manually tested?

- Clone the repo
- checkout to branch `ft-sign-in-8`
- pull the changes by `git pull`
- install all packages by running `npm i`
- start app by running `npm run start:dev`

**In your API testing tool (postman recommended) call API** 
- `method: Post` `http://localhost:3001/api/v1/auth/login` to be able to login with email and password

#### Any background context you want to provide?

- Note that you should send the request with email and password to be able to login
- **request body** 
 ```js script
  {
   "email":"test@gmail.com",
   "password":"password@123"
 }
  ```

#### What are the relevant Pivotal Tracker/Trello stories?

- https://trello.com/c/VSqRH6bg/8-sign-in-to-phantom

#### Screenshots (if appropriate)

-  N/A

#### Questions:

-  N/A

### CODE REVIEW

File: src/tests/app.test.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+//       it('it should GET the homepage', (done) => {
+//         chai.request(app)
+//             .get('/')
+//             .end((err, res) => {
+//               res.should.have.status(200);
+//               // res.status.should.be.equal(200);
+//               done();
+//             });
+//       });
+//   });
```
Comment (thepatrickniyo) [inline src/tests/app.test.js]: Let's remove these

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
GLTM though you have to remove those commented source codes for the best quality

File: src/tests/app.test.js (hunk 1 above)
Comment (PrinceNiyonshuti) [reply src/tests/app.test.js]: yes noted

**Review by Xldivin — APPROVED** [review]

**Review by Hodal66 — APPROVED** [review]

**Review by mukunzidd — APPROVED** [review]
LGTM.

**Review by tysepa — APPROVED** [review]

**Review by thepatrickniyo — APPROVED** [review]

**Pull request merged**